### DISCUSSION TITLE
( Sub Dynamic page  features ) I will recommended coming soon version, you will update modify

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5645] [New Feature/Ideas Requests]
When I press on a banner or button from the homepage the hidden dynamic page will show.

I can get new tab bar in that hidden dynamic page

Follow this video -

https://streamable.com/eu0313
https://streamable.com/9nnvzx

**Comment by ameenbadri7** [comment]
on your config_en.json file, you can make the banner go to the Brand page, categories page, products, and anything you want.
